tools/cmake/tests/mbed_test_mode/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../.. CACHE INTERNAL "")
+set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE STATIC "")
+
+include(${MBED_PATH}/tools/cmake/app.cmake)
+
+add_executable(mbed-test-mode-check)
+project(mbed-test-mode-check)
+
+set(PROJECT_NAME ${CMAKE_PROJECT_NAME} CACHE INTERNAL "")
+include(CTest)
+
+target_sources(mbed-test-mode-check
+    PRIVATE
+        main.cpp
+)
+
+add_subdirectory(
+    ${MBED_PATH} build
+)
```

Comment:
Where is this "build" directory?

Reply:
Do we need to `add_subdirectory()` at all?